[bitnami/keycloak] with MYSQL still connects to PostgreSQL

### Name and Version

bitnami/keycloak-9.4.2

### What steps will reproduce the bug?

K3s Setup.


### Are you using any custom parameters or values?
```
helm install \
--namespace=default \
--timeout=10m0s \
--version=7.0.3 \
--set extraEnvVarsCM=ef-keycloak-config  \
--set auth.adminPassword=admin  \
--set auth.managementPassword=admin  \
--set auth.adminUser=admin  \
--set image.debug=true \
--set service.type=ClusterIP \
--set postgresql.enabled=false \
--set externalDatabase.host=mysql.default.svc.cluster.local \
--set externalDatabase.port=3306 \
--set externalDatabase.user="keycloak_user" \
--set externalDatabase.password="Password464" \
--set externalDatabase.database="keycloak_db" \
--set extraEnvVars[0].name=DB_VENDOR \
--set extraEnvVars[0].value=MYSSQL \
keycloak bitnami/keycloak
```
### What is the expected behavior?

ideally keycloak should be able to connect to the MySQL DB but its not.

### What do you see instead?

It should be using the MySQL database driver, however, in logs I can see 

```console
keycloak 18:21:50.88 INFO  ==> ** Starting keycloak setup **
keycloak 18:21:50.92 INFO  ==> Validating settings in KEYCLOAK_* env vars...
keycloak 18:21:50.97 INFO  ==> Trying to connect to PostgreSQL server mysql.default.svc.cluster.local...
keycloak 18:21:51.01 INFO  ==> Found PostgreSQL server listening at mysql.default.svc.cluster.local:3306
keycloak 18:21:51.06 INFO  ==> Configuring database settings
You are disconnected at the moment. Type 'connect' to connect to the server or 'help' for the list of supported commands.
```

```
[[standalone@embedded /] batch
[standalone@embedded / #] /subsystem=datasources/data-source=KeycloakDS: remove()
ccm=true, connection-url=jdbc:postgresql://mysql.default.svc.cluster.local:3306/keycloak_db, driver-name=postgresql)akDS,enabled=true,use-java-context=true,use-
[standalone@embedded / #] /subsystem=datasources/data-source=KeycloakDS: write-attribute(name=user-name, value=${env.KEYCLOAK_DATABASE_USER})
[standalone@embedded / #] /subsystem=datasources/data-source=KeycloakDS: write-attribute(name=check-valid-connection-sql, value="SELECT 1")
[standalone@embedded / #] /subsystem=datasources/data-source=KeycloakDS: write-attribute(name=background-validation, value=true)
[standalone@embedded / #] /subsystem=datasources/data-source=KeycloakDS: write-attribute(name=background-validation-millis, value=60000)
[standalone@embedded / #] /subsystem=datasources/data-source=KeycloakDS: write-attribute(name=flush-strategy, value=IdleConnections)
-class-name=org.postgresql.xa.PGXADataSource)ces/jdbc-driver=postgresql:add(driver-name=postgresql, driver-module-name=org.postgresql.jdbc, driver-xa-datasource
[standalone@embedded / #] /subsystem=keycloak-server/spi=connectionsJpa/provider=default:write-attribute(name=properties.schema,value=public)
[standalone@embedded / #10969 
```
```
18:27:53,703 INFO  [org.jboss.as.connector.subsystems.datasources] (ServerService Thread Pool -- 32) WFLYJCA0004: Deploying JDBC-compliant driver class org.h2.Driver (version 1.4)
18:27:53,829 INFO  [org.jboss.as.connector.subsystems.datasources] (ServerService Thread Pool -- 32) WFLYJCA0005: Deploying non-JDBC-compliant driver class org.postgresql.Driver (version 42.2)
18:27:53,904 WARN  [org.wildfly.clustering.web.undertow] (ServerService Thread Pool -- 54) WFLYCLWEBUT0007: No routing provider found for default-server; using legacy provider based on static 
        at io.undertow.servlet@2.2.14.Final//io.undertow.servlet.core.DeploymentManagerImpl.start(DeploymentManagerImpl.java:601)
        at org.wildfly.extension.undertow@26.0.0.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentService.startContext(UndertowDeploymentService.java:106)
        at org.wildfly.extension.undertow@26.0.0.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentService$1.run(UndertowDeploymentService.java:87)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1990)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1486)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1377)
        at java.base/java.lang.Thread.run(Thread.java:829)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.JBossThread.run(JBossThread.java:513)
Caused by: org.postgresql.util.PSQLException: An error occurred while setting up the SSL connection.
        at org.postgresql.jdbc@42.2.14//org.postgresql.core.v3.ConnectionFactoryImpl.enableSSL(ConnectionFactoryImpl.java:450)
        at org.postgresql.jdbc@42.2.14//org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:140)
        at org.postgresql.jdbc@42.2.14//org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:197)
        at org.postgresql.jdbc@42.2.14//org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
        at org.postgresql.jdbc@42.2.14//org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:217)
        at org.postgresql.jdbc@42.2.14//org.postgresql.Driver.makeConnection(Driver.java:458)
        at org.postgresql.jdbc@42.2.14//org.postgresql.Driver.connect(Driver.java:260)
        at org.jboss.ironjacamar.jdbcadapters@1.5.3.Final//org.jboss.jca.adapters.jdbc.local.LocalManagedConnectionFactory.createLocalManagedConnection(LocalManagedConnectionFactory.java:335)
        ... 60 more

18:28:09,690 FATAL [org.keycloak.services] (ServerService Thread Pool -- 66) Error during startup: java.lang.RuntimeException: Failed to connect to database
```

### Additional information
```
[root@devops230 EF-CIM-SR7]# more pre-deployment/ef-keycloak-configmap.yaml
apiVersion: v1
data:
  KEYCLOAK_ADMIN_URL: https://devops233.com/auth/realms/myrealm/account
  KEYCLOAK_EXTRA_ARGS: -Dkeycloak.frontendUrl=https://devops233.com/auth  -Dkeycloak.profile.feature.upload_scripts=enabled
  KEYCLOAK_FRONTEND_URL: https://devops233.com/auth/
  KEYCLOAK_LOGLEVEL: DEBUG
  KEYCLOAK_PASSWORD: admin
  KEYCLOAK_PROXY_ADDRESS_FORWARDING: "true"
  KEYCLOAK_USER: admin
  NODE_ENV: development
  DB_VENDOR: MYSQL
  JAVA_OPTS_APPEND: "-Djboss.as.management.blocking.timeout=7200"
kind: ConfigMap
metadata:
  name: ef-keycloak-config
```

--- a/bitnami/keycloak/README.md
+++ b/bitnami/keycloak/README.md
@@ -7,7 +7,7 @@ Keycloak is a high performance Java-based identity and access management solutio
 [Overview of Keycloak](https://www.keycloak.org/)
 
 Trademarks: This software listing is packaged by Bitnami. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
-                           
+
 ## TL;DR
 
 ```console
@@ -359,9 +359,11 @@ Bitnami will release a new chart updating its containers if a new version of the
 
 ### Use an external database
 
-Sometimes, you may want to have Keycloak connect to an external database rather than a database within your cluster - for example, when using a managed database service, or when running a single database server for all your applications. To do this, set the `postgresql.enabled` parameter to `false` and specify the credentials for the external database using the `externalDatabase.*` parameters.
+Sometimes, you may want to have Keycloak connect to an external PostgreSQL database rather than a database within your cluster - for example, when using a managed database service, or when running a single database server for all your applications. To do this, set the `postgresql.enabled` parameter to `false` and specify the credentials for the external database using the `externalDatabase.*` parameters.
 
 Refer to the [chart documentation on using an external database](https://docs.bitnami.com/kubernetes/apps/keycloak/configuration/use-external-database) for more details and an example.
+
+> NOTE: Only PostgreSQL database server is supported as external database
 
 ### Add extra environment variables
 

